### DISCUSSION TITLE
error importing an epub

### DISCUSSION THREAD

**Discussion by theob1966** [knowledgefutures/pubpub#1920] [Support]
Hi there- can anyone indicate what the error might be?
I'm trying to add thr epub version of the Immunsation handbook from this site: https://www.health.govt.nz/publication/immunisation-handbook-2020

when I do so, the following happens:

**Upload epub  file is successful

Importing document**

Import warnings:
•	Your document contains references external images which you may wish to upload.
•	Missingimages
•data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMTAwJSIgcHJlc2VydmVhc3BlY3RyYXRpbz0ieE1pZFlNaWQgbWVldCIgdmVyc2lvbj0iMS4xIiB2aWV3Ym94PSIwIDAgNTYwIDc5MiIgd2lkdGg9IjEwMCUiIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIj4KCjwvc3ZnPg==.
•	Conversion from LaTeX to HTML succeeded, with information ([more about PubPub and LaTeX](https://help.pubpub.org/pub/latex-compatibility)):
[WARNING] Duplicate identifier 'sigil_toc_id_191'

Click ‘complete import’

Pop up says: 
An error has occured in the editor.
We've logged the error and will look into the cause right away.
All previous changes have been successfully saved.
To continue editing, please refresh the page.

I refresh page and the epub has not been imported

cheers Theo

**Comment by gabestein** [comment]
Hi Theo – sorry about the delay here. It looks like there's an issue with the way a particular bulleted list, of all things, is being handled in-between the conversion tool we use (pandoc) and our internal editor schema. I'm going to convert this to an issue, and we'll take a look at it next week.

In the meantime, I was able to convert the epub to markdown, and import that successfully. I've attached the markdown here (as a text file) if you'd like to use it. It may have lost some fidelity from the epub, so I don't know if it'll work for you, but I wanted to give you an option in case it does allow you to move forward.

[immune.md](https://github.com/pubpub/pubpub/files/8399712/immune.md)

**Comment by theob1966** [comment]
Thanks Gabriel- getting back to this now- the md doesn’t quite work.
Having looked the community service page- I am wondering if a collaborative approach with pubpub is the best way to go instead?
